pkg/server/commands/reverseexpand/reverse_expand_test.go

```diff
@@ -144,4 +144,120 @@ type document
 	case <-done:
 		// OK!
 	}
+
+	select {
+	case <-done:
+		t.Log("OK!")
```

Comment:
Do we need this residual `t.Log`? Seems too non-specific to be worth keeping.

Reply:
cleaned up